### ISSUE TITLE
React Developer Tools 

### ISSUE THREAD

**Issue by erikwatson** [MarshallOfSound/electron-devtools-installer#149]
Many of the extensions seem to work with Electron 9 and 10, but the React Developer Tools simply do not start up on those versions. I had to downgrade to Electron 8.5.1 before it would work.

This should at least be mentioned in the Readme. It's been quite frustrating.

**Comment by ericblade** [comment]
I get 
````
(node:5008) ExtensionLoadWarning: Warnings loading extension at C:\Users\Eric\AppData\Roaming\Electron\extensions\fmkadmapgofadopljbjfkapdkoienihi: Unrecognized manifest key 'browser_action'. Unrecognized manifest key 'minimum_chrome_version'. Unrecognized manifest key 'update_url'. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
````
but it never resolves *or* rejects the install request for REACT_DEVELOPER_TOOLS.

I do get a similar error when asking for REDUX_DEVTOOLS, but it succeeds:

````
(node:5008) ExtensionLoadWarning: Warnings loading extension at C:\Users\Eric\AppData\Roaming\Electron\extensions\lmhkpmbekcpmknklioeibfkpmmfibljd: Unrecognized manifest key 'commands'. Unrecognized manifest key 'homepage_url'. Unrecognized manifest key 'page_action'. Unrecognized manifest key 'short_name'. Unrecognized manifest key 'update_url'. Permission 'notifications' is unknown or URL pattern is malformed. Permission 'contextMenus' is unknown or URL pattern is malformed. Permission 'tabs'
6:26:50 AM electron.1 |  >  is unknown or URL pattern is malformed. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
6:26:50 AM electron.1 |  Added extension Redux DevTools
````

**Comment by ericblade** [comment]
also, found this on the console of the renderer browser view:

![image](https://user-images.githubusercontent.com/1451847/97863649-51262c00-1cd5-11eb-8034-80a117634f7d.png)

**Comment by abramsimon** [comment]
Not optimal, but I was able to work around this by removing the `_manifest` folder for the extension. 

On MacOS I found the folder at 
```
~/Library/Application Support/my-electron-app/extensions/fmkadmapgofadopljbjfkapdkoienihi/_metadata
```

It still throws some warnings, but the extension loads and I'm able to see my components and state.
```
(node:86623) ExtensionLoadWarning: Warnings loading extension at /Users/me/Library/Application Support/my-electron-app/extensions/fmkadmapgofadopljbjfkapdkoienihi: Unrecognized manifest key 'browser_action'. Unrecognized manifest key 'minimum_chrome_version'. Unrecognized manifest key 'update_url'. 
```

**Comment by ericblade** [comment]
Removing the _metadata gets rid of that error message, but it still doesn't install the React tools.  I *do* get the same error message with the Redux tools, but it does successfully load.

Interestingly, the console eventually fills up with messages that look like this: 

![image](https://user-images.githubusercontent.com/1451847/97918547-48107b80-1d24-11eb-9033-5aac1df80863.png)

... note that ddjeflhhnbddgec ... is *not* the same hash id as the archive, which is fmkadmapgofadopljbjfkapdkoienihi

**Comment by ericblade** [comment]
... I would like to retract much of what I just said above -- The dev tools looks *completely* different than it did last time I used it.  I'm not sure when it started working, but I didn't even recognize it.  Last time I used it it was obvious which tabs were the dev tools, now it's just .. not. lol. 

Still, my error messages that i've detailed above are all still happening.

**Comment by aguynamedben** [comment]
My warnings, in case it's helpful.

```
hrome_version'. Unrecognized manifest key 'update_url'. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
(node:87444) ExtensionLoadWarning: Warnings loading extension at /Users/ben/Library/Application Support/Electron/extensions/lmhkpmbekcpmknklioeibfkpmmfibljd: Unrecognized manifest key 'commands'. Unrecognized manifest key 'homepage_url'. Unrecognized manifest key 'page_action'. Unrecognized manifest key 'short_name'. Unrecognized manifest key 'update_url'. Permission 'notifications' is unknown or URL pattern is malformed. Permission 'contextMenus' is unknown or URL pattern is malformed. Permission 'tabs' is unknown or URL pattern is malformed. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
debug: [main] creating windows
```

I also have the issue where Redux devtools show, but not React devtools.

I've tried `rm -rf ~/Library/Application\ Support/Electron/extensions` and running installExtension with `forceDownload=true`, and that doesn't fix it.

**Comment by MarshallOfSound** [comment]
Hey folks, appreciate your usage of `electron-devtools-installer`. We're going through a bit of spring cleaning and closing out all the old issues that haven't been looked at.

**This doesn't mean I'm ignoring you, it just means I don't have the time to catch up on 4 years of GitHub issue backlog**

If this issue is still impacting you on the **latest** release of `electron-devtools-installer` and a **supported** release of Electron (see https://releases.electronjs.org) please raise a new issue with a solid repro case and detailed instructions.
